src/UID2.Client/UID2Encryption.cs

```diff
@@ -93,16 +94,21 @@ private static DecryptionResponse DecryptV2(byte[] encryptedId, KeyContainer key
 
             if (expiry < now)
             {
-                return new DecryptionResponse(DecryptionStatus.ExpiredToken, null, established, siteId, siteKey.SiteId);
+                return new DecryptionResponse(DecryptionStatus.ExpiredToken, null, established, siteId, siteKey.SiteId, null, 2);
             }
             else
             {
-                return new DecryptionResponse(DecryptionStatus.Success, idString, established, siteId, siteKey.SiteId);
+                return new DecryptionResponse(DecryptionStatus.Success, idString, established, siteId, siteKey.SiteId, null, 2);
             }
         }
 
-        private static DecryptionResponse DecryptV3(byte[] encryptedId, KeyContainer keys, DateTime now, IdentityScope identityScope)
+        private static DecryptionResponse DecryptV3(byte[] encryptedId, KeyContainer keys, DateTime now, IdentityScope identityScope, IdentityType? identityType, int advertisingTokenVersion)
         {
+            if (identityType == null)
```

Comment:
@jon8787 just wanna do a sanity check - as we previously fixed the codes in SDKs back in ~April as the Identity Type prefix wasn't correct. So if the identityTYpe/GetIdentityType below aren't matched to A/B/E/F we would return error response here. 

So my concern is what if people still have tokens which the wrong prefix and get passed into here and we reject it (even tho it's a legit ad token but with wrong prefix). But this isn't a concern because we haven't enabled v4 token in bidstream and also it was only SDK-created sharing token would have the wrong prefix previously so therefore this is not a concern?

Reply:
per slack, let's not reject tokens just because we couldn't figure out the identity type